src/components/filters/FiltersButton.tsx

```diff
@@ -0,0 +1,46 @@
+import sortIcon from "../../assets/sort-icon.svg";
+import chevronDownIcon from "../../assets/chevron-down.svg";
+import xIcon from "../../assets/x-lg.svg";
+import { Dispatch, useState } from "react";
+import Filters from "./FilterList";
+import { Actions } from "../../reducers/productsReducer";
+interface FiltersButtonProps {
+  dispatch: Dispatch<Actions>;
+}
+const FiltersButton = ({ dispatch }: FiltersButtonProps) => {
+  const [collapsed, setCollapsed] = useState(false);
+
+  const handleSetCollapsed = (collapsed: boolean) => {
+    setCollapsed(collapsed);
+  };
+
+  return (
+    <div className="filters-wrapper">
+      <button
+        className="btn filters-button"
+        onClick={() => handleSetCollapsed(true)}
+      >
+        <img src={sortIcon} alt="" />
+        All filters
+        <img src={chevronDownIcon} alt="" />
+      </button>
```

Comment:
Specify alt attributes

Reply:
Those icons are only decorative, "All filters" determine what this button does.